docs/en/Running-Alluxio-on-a-Cluster.md

```diff
@@ -200,6 +200,31 @@ Alluxio leader master.
 hadoop fs -ls alluxio:///directory
 ```
 
+##### Alluxio on Zookeeper URI
+
+Set `alluxio.zookeeper.enable` and `alluxio.zookeeper.address` is always favored when client applications
```

Comment:
so we still recommend to use `alluxio.zookeeper.enable`?

Reply:
alluxio://zk@ is not mature and still need more testing. In many circumstances, I think setting configuration is still better(but I am not sure what circumstances). By now, I just mention "alluxio://zk@" as tips in the docs of computation frameworks. We could discuss later if we want to make it a recommended way for our users.